Add images and tests for JDBC Sink, ES Sink and Replicator

arrawatia opened the pull request.

theduderog commented on debian/base/Dockerfile:
```diff
@@ -31,14 +31,18 @@ ENV PYTHON_PIP_VERSION="8.1.2"
+ENV CONFLUENT_DEB_REPO="https://s3-us-west-2.amazonaws.com/staging-confluent-packages-3.1.0-snapshot"
+
+# REMOVE SNAPSHOT
+ENV KAFKA_VERSION="0.10.1.0~SNAPSHOT"
```
why define this in the base now?

theduderog commented:
LGTM other than the tearDownClass stuff that seems to be commented out by mistake.  @samjhecht, are you good?  if so, i can fix the teardown() and merge.

arrawatia commented on tests/test_kafka_connect.py:
```diff
@@ -206,10 +220,10 @@ def setUpClass(cls):
-    def tearDownClass(cls):
-        cls.machine.ssh("sudo rm -rf /tmp/kafka-connect-single-node-test")
-        cls.cluster.shutdown()
+    # @classmethod
```
Yes. I fixed it in the latest commit.

arrawatia commented:
@theduderog I fixed that and also created a base connect image and renamed the replicator image to enterprise-replicator. Can you please review these changes ?

theduderog commented:
your still awake!  LGTM

arrawatia commented:
@theduderog Thanks.
